### DISCUSSION TITLE
Stuck with failed checks of Vercel in my PR.

### DISCUSSION THREAD

**Discussion by agrittiwari** [vercel/vercel#8337] [Help] [answered]
I have an approved PR, but these checks keep failing and won't let my PR get merge, even after enabling auto-merge. Why are these checks failing and wat can I do to resolve them?

Link to the [PR](https://github.com/supabase/supabase/pull/7989)
need help!

![image](https://user-images.githubusercontent.com/51356497/183256029-4d5c882f-099e-4efc-8719-790f1949b254.png)

**Comment by leerob** [comment, marked as answer]
The Supabase team needs to authorize the deployment first. 

<img width="701" alt="CleanShot 2022-08-06 at 15 23 06@2x" src="https://user-images.githubusercontent.com/9113740/183264973-e54b2dd6-392b-4f10-9a1a-2e4ae41a389c.png">
